hack/lib.sh

```diff
@@ -396,7 +405,7 @@ start_docker_daemon_ci() {
 }
 
 start_docker_daemon() {
-  if docker stats --no-stream > /dev/null 2>&1; then
+  if $CONTAINER_RUNTIME stats --no-stream > /dev/null 2>&1; then
     echodate "Not starting Docker again, it's already running."
     return
   fi
```

Comment:
I don't think it makes sense to replace the `docker` calls in this function, because this is explicitly to start docker. Did you run into this during your development or is it a simple search and replace?

Reply:
fixed